src/jupyter_contrib_nbextensions/nbextensions/execute_time/ExecuteTime.js

```diff
@@ -269,6 +270,9 @@ define([
             var exec_time = -start_time.diff(end_time);
             msg = msg.replace('${duration}', humanized_duration(exec_time));
         }
+        if (exec_time < options.hide_shorter_than_threshold_in_ms ) {
```

Comment:
`exec_time` is potentially undefined here. Although that works ok, because `undefined < any_int == false`, it's not clear that that's the intent. Better might be something like

```js
var show_timing = true;
if (end_time) {
    end_time = moment(end_time);
    msg = msg.replace('${end_time}', format_moment(end_time));
    var exec_time = -start_time.diff(end_time);
    msg = msg.replace('${duration}', humanized_duration(exec_time));
    show_timing = exec_time >= options.hide_shorter_than_threshold_in_ms;
}
toggle_timing_display([cell], show_timing);
```

Reply:
Good point. Thanks for the suggestion. Updated the code.